pcsx2/USB/usb-pad/usb-pad-ff.cpp

```diff
@@ -98,11 +98,11 @@ namespace usb_pad
 	{
 		if (!mFFdev)
 			return;
-
+#ifdef DEBUG
```

Comment:
might be better to have `#if defined(PCSX2_DEVBUILD) || defined(_DEBUG)`

Reply:
Fixed